setup.py

```diff
@@ -19,6 +19,10 @@
     ],
     "dev": dev_requires + test_requires,
     "dropbox": ["dropbox ~= 9.0"],
+    "gcp": [
+        "google-cloud-bigquery >= 1.6.0, < 2.0",
+        "google-cloud-storage >= 1.13, < 2.0",
+    ],
     "google": [
         "google-cloud-bigquery >= 1.6.0, < 2.0",
         "google-cloud-storage >= 1.13, < 2.0",
```

Comment:
I think `google` can be removed

Reply:
I agree

Reply:
Yea but that would be a breaking change for requirements files; I figured we would remove that when we remove the analogously changed submodule

Reply:
@jlowin / @joshmeek let me know if you agree; I can remove it if y'all feel strongly.